spec/calculator.spec.js

```diff
@@ -16,288 +16,304 @@
-      instance.addNumber(6);
-      expect(instance.input()).to.equal("6");
-      instance.addNumber(2);
+    it('It should be able to perform an add operation with more than single digit numbers', () => {
+      instance.addNumber('6');
+      instance.addNumber('2');
+      instance.addPlus();
+      instance.addNumber('3');
+      instance.addNumber('2');
+      instance.equal();
+      expect(instance.input()).to.equal("94");
+    });
+
+    it('It should display the currently entered number when more than 1 digit is entered', () => {
+      instance.addNumber('6');
+      instance.addNumber('2');
       expect(instance.input()).to.equal("62");
+    });
+
+    it('It should continue to display the first operand after an operation has been pressed and the user has not yet started typing the second operand', () => {
+      instance.addNumber('6');
+      instance.addNumber('2');
       instance.addPlus();
       expect(instance.input()).to.equal("62");
-      instance.addNumber(3);
-      expect(instance.input()).to.equal("3");
-      instance.addNumber(2);
+    });
+
+    it('It should display only the second operand when an operation has been pressed and the user is typing the second operand', () => {
+      instance.addNumber('6');
+      instance.addNumber('2');
+      instance.addPlus();
+      instance.addNumber('3');
+      instance.addNumber('2');
       expect(instance.input()).to.equal("32");
-      instance.equal();
-      expect(instance.input()).to.equal(94);
     });
-// can i do something? yep
-// done 
-    it('It should be able to perform a subtract operation', () => {
-      instance.addNumber(1);
-      expect(instance.input()).to.equal("1");
-      instance.addNumber(4);
-      expect(instance.input()).to.equal("14");
+
+
+    it('It should be able to perform an subtract operation with more than single digit numbers', () => {
+      instance.addNumber('2');
+      instance.addNumber('2');
       instance.addMinus();
-      expect(instance.input()).to.equal("14");
-      instance.addNumber(1);
-      expect(instance.input()).to.equal("1");
-      instance.addNumber(0);
-      expect(instance.input()).to.equal("10");
+      instance.addNumber('1');
+      instance.addNumber('2');
       instance.equal();
-      expect(instance.input()).to.equal(4);
+      expect(instance.input()).to.equal("10");
     });
 
-    it('It should be able to perform a subtract operation', () => {
-      instance.addNumber(1);
-      expect(instance.input()).to.equal("1");
-      instance.addNumber(4);
-      expect(instance.input()).to.equal("14");
+    it('It should display the currently entered number when more than 1 digit is entered', () => {
+      instance.addNumber('2');
+      instance.addNumber('2');
+      expect(instance.input()).to.equal("22");
+    });
+
+    it('It should continue to display the first operand after an operation has been pressed and the user has not yet started typing the second operand', () => {
```

Comment:
This test has the same name as the one for addPlus. You should rename "after an operation has been pressed" to "after subtract has been pressed"